integration-test/src/test/java/com/datastrato/gravitino/integration/test/spark/iceberg/SparkIcebergCatalogIT.java

```diff
@@ -216,6 +233,24 @@ void testIcebergMetadataColumns() throws NoSuchTableException {
     testDeleteMetadataColumn();
   }
 
+  @Test
+  void testInjectSparkExtensions() {
```

Comment:
seems no need to add this test? since we had  Iceberg specific tests.

Reply:
ok

Reply:
removed.